scipy/special/cephes/ellik.c

```diff
@@ -79,78 +84,88 @@ double ellik(double phi,  double m)
             return NPY_NAN;
     }
     if (m == 0.0)
-	return (phi);
+        return (phi);
     a = 1.0 - m;
-    if (a == 0.0) {
-	if (fabs(phi) >= NPY_PI_2) {
-	    mtherr("ellik", SING);
-	    return (NPY_INFINITY);
-	}
+    if (a == 0.0)
+    {
+        if (fabs(phi) >= NPY_PI_2)
+        {
+            mtherr("ellik", SING);
+            return (NPY_INFINITY);
+        }
         /* DLMF 19.6.8, and 4.23.42 */
-       return asinh(tan(phi));
+        return asinh(tan(phi));
     }
     npio2 = floor(phi / NPY_PI_2);
     if (fmod(fabs(npio2), 2.0) == 1.0)
-	npio2 += 1;
-    if (npio2 != 0.0) {
-	K = ellpk(a);
-	phi = phi - npio2 * NPY_PI_2;
+        npio2 += 1;
+    if (npio2 != 0.0)
+    {
```

Comment:
WRT the python files, they were not in compliance with PEP8. WRT the cpp files, you appear to be correct regarding the conditional blocks. I will attempt to resolve that.